### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by phoracek**
With this PR we bump dependencies to the latest release of containernetworking/ and vishvananda, Go 1.13 and Kubernetes 1.18.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,42 +1,50 @@
 module github.com/kubevirt/ovs-cni
 
-go 1.12
```
Comment (qinqon) [inline go.mod:3]: go 1.13

Comment (phoracek) [reply go.mod:3]: It is added at the bottom of the file

**Comment by phoracek** [comment]
> Commit ‘ update pinned dependencies’: why did 924 file changed?

We re-fetched all libraries we depend on that are not explicitly pinned to a certain version.

> How can this be safely reviewed?

It can't. We trust tests.

**Pull request merged**